### PR TITLE
onfluent-platform: fix confluent-platform cli

### PR DESCRIPTION
**Pull request by bozzzzo**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
Confluent-platform install a `confluent` CLI that is not really working.
Two fixes are needed:
- install the `libexec` folder
- fix the `confluent` script to find the keg when run from `/usr/local/bin`

### CODE REVIEW

File: Formula/confluent-platform.rb [hunk 1]
```diff
@@ -11,14 +11,36 @@ class ConfluentPlatform < Formula
...
     prefix.install "share"
+    prefix.install "libexec"
+    rm_rf "#{libexec}/cli/linux_386"
+    rm_rf "#{libexec}/cli/linux_amd64"
+    rm_rf "#{libexec}/cli/windows_386"
+    rm_rf "#{libexec}/cli/windows_amd64"
   end
 
   test do
     assert_match version.to_s, shell_output("#{bin}/kafka-broker-api-versions --version")
```
Comment (SMillerDev) [inline Formula/confluent-platform.rb]: Could you improve this test to use some of the functionality of the software? Version tests rarely indicate that something is actually wrong. And especially if you're going to patch something we need a test.

**Review by SMillerDev — COMMENTED** [review]
.

File: Formula/confluent-platform.rb [hunk 2]
```diff
@@ -11,14 +11,36 @@ class ConfluentPlatform < Formula
 
   conflicts_with "kafka", :because => "kafka also ships with identically named Kafka related executables"
 
+  patch :DATA
```
Comment (zbeekman) [inline Formula/confluent-platform.rb]: Has this been reported upstream? This needs a comment and a link to an upstream issue at a minimum and description of when it should be removed or how it should be updated. In an ideal world, upstream would accept the fix and we would use a `patch do` block to grab the patch from github.

**Comment by satyamagarwal** [comment]
I recently installed confluent-platform and I got this error : 
`confluentinc/cli crit executable /usr/local/bin/../libexec/cli/darwin_amd64/confluent does not exist.  Make sure this script is up-to-date and resides at bin/confluent in the Confluent Platform release directory.` 

I think this pull request talks exactly about this ?

**Comment by bozzzzo** [comment]
> I think this pull request talks exactly about this ?

Yepyep.

Right now I don't have the time to do all the polishing required for this to be approved.
There is one further tweak missing, which is to do: 

        export CONFLUENT_HOME=/usr/local/opt/confluent-platform

I believe there is a workaround, try running it as `/usr/local/opt/confluent-platform/bin/confluent`. If this works you can just put that path in front of `/usr/local/bin`, but it's clumsy.

**Pull request closed without merging**